test_phage_reference/1_build_reference_transcriptome.ipynb

```diff
@@ -41,19 +41,383 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
+   "execution_count": 2,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Version Info: ### PLEASE UPGRADE SALMON ###\n",
+      "### A newer version of salmon with important bug fixes and improvements is available. ####\n",
+      "###\n",
+      "The newest version, available at https://github.com/COMBINE-lab/salmon/releases\n",
+      "contains new features, improvements, and bug fixes; please upgrade at your\n",
+      "earliest convenience.\n",
+      "###\n",
+      "Sign up for the salmon mailing list to hear about new versions, features and updates at:\n",
+      "https://oceangenomics.com/subscribe\n",
+      "###\n",
+      "[2020-12-22 10:50:46.910] [jLog] [warning] The salmon index is being built without any decoy sequences.  It is recommended that decoy sequence (either computed auxiliary decoy sequence or the genome of the organism) be provided during indexing. Further details can be found at https://salmon.readthedocs.io/en/latest/salmon.html#preparing-transcriptome-indices-mapping-based-mode.\n",
+      "[2020-12-22 10:50:46.910] [jLog] [info] building index\n",
+      "out : /home/alexandra/Documents/Data/Core_accessory/pao1_index\n",
+      "\u001b[00m[2020-12-22 10:50:46.911] [puff::index::jointLog] [info] Running fixFasta\n",
+      "\u001b[00m\n",
+      "[Step 1 of 4] : counting k-mers\n",
+      "\n",
+      "\u001b[35m[2020-12-22 10:50:47.079] [puff::index::jointLog] [warning] Removed 34 transcripts that were sequence duplicates of indexed transcripts.\n",
```

Comment:
I'm not sure, I haven't spent too much time looking at these outputs and more the expression data. This would probably be helpful just to have more of a sense for how Salmon is working